index-map-area.html

```diff
@@ -36,6 +36,14 @@
       map[is="web-map"]:not(:defined) > :not(area):not(.mapml-web-map) {
         display: none;
       }
+      
+      /* Pre-style to avoid Layout Shift. */
+      map[is="web-map"]:not(:defined) {
+        display: inline-block;
+        contain: size;
+        contain-intrinsic-size: 304px 154px;
```

Comment:
Just a note:

`contain-intrinsic-size: 304px 154px` may look strange, one might expect `300px 150px` (same as the default size of `width: 300px` and `height: 150px`), it is used because the map viewers have a 2px `border-width`.

I.e. `contain-intrinsic-size: 300px 150px` would still cause a small but noticable layout shift.

Reply:
Nice work!